packages/components/src/components/radio-button/radio-button.ts

```diff
@@ -166,51 +171,47 @@ export default class SdRadioButton extends SolidElement {
       }
 
       .lg-no-label {
-        @apply h-12 w-12;
+        @apply h-12 px-5;
       }
 
       .md-label {
         @apply h-10;
       }
 
       .md-no-label {
-        @apply h-10 w-10;
+        @apply h-10 px-4;
       }
 
       .sm-label {
         @apply h-8;
       }
 
       .sm-no-label {
-        @apply h-8 w-8;
-      }
-
-      :host(.sd-button-group__button--first:not(.sd-button-group__button--last)) button {
-        @apply rounded-r-none;
+        @apply h-8 px-3;
       }
 
-      :host(.sd-button-group__button--inner) button {
-        @apply rounded-none;
+      /* All except the first */
+      :host([size='lg'].sd-button-group__button:not(.sd-button-group__button--first)) {
+        @apply -ml-3;
       }
 
-      :host(.sd-button-group__button--last:not(.sd-button-group__button--first)) button {
-        @apply rounded-l-none;
+      /* All except the first */
+      :host([size='md'].sd-button-group__button:not(.sd-button-group__button--first)) {
+        @apply -ml-2;
       }
 
       /* All except the first */
-      :host(.sd-button-group__button:not(.sd-button-group__button--first)) {
-        margin-inline-start: -1px;
+      :host([size='sm'].sd-button-group__button:not(.sd-button-group__button--first)) {
+        @apply -ml-1;
```

Comment:
We can use `-ms-1`, which will be the same as `margin-inline-start`. This way `rtl` is still supported

```suggestion
        @apply -ms-1;
```